Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/ui/DebugLogFragment.kt

```diff
@@ -138,4 +150,52 @@ class DebugLogFragment : Fragment(R.layout.bugreporting_debuglog_fragment), Auto
             if (resultCode == Activity.RESULT_OK) resultData?.data else null
         )
     }
+
+    private fun showLogDeletionConfirmation() {
+        AlertDialog.Builder(requireContext()).apply {
+            setMessage(R.string.debugging_debuglog_stop_confirmation_message)
+            setPositiveButton(android.R.string.yes) { _, _ -> }
```

Comment:
what happens in Yes is clicked?

Reply:
Nothing. The dialog is closed. The text reads "OK" despite being called "yes".

I found it a bit confusing, as I thought the confirmation is to prevent accidental deletion, but afaik the ticket wants this to be just a "post-fact" information dialog. Right?

Reply:
According to figma: I think it just dismisses the dialog.